File: src/main/java/es/uji/apps/hor/model/Curso.java
package es.uji.apps.hor.model;

public class Curso
{
    private Long curso;

    public Curso(Long curso)
    {
        this.setCurso(curso);
    }

    public Long getCurso()
    {
        return curso;
    }

    public void setCurso(Long curso)
    {
        this.curso = curso;
    }

}

File: trunk/src/main/webapp/app/model/Curso.js
Ext.define('HOR.model.Curso',
{
    extend : 'Ext.data.Model',

    fields : [ 'id', 'curso' ],
    validations : [
    {
        type : 'length',
        field : 'id',
        min : 1
    } ]
});